Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
+COPY . .
+CMD npm run start
```

Comment:
I would write `CMD [ "npm", "run", "dev" ]`

Reply:
I've run my project in docker with this config

Reply:
`CMD ["executable","param1","param2"] (exec form, this is the preferred form)
CMD ["param1","param2"] (as default parameters to ENTRYPOINT)
CMD command param1 param2 (shell form)`

Reply:
Yes, I also tried and it's work